app/components/ImagesList.js

```diff
@@ -1,17 +1,16 @@
 import React from 'react';
 import Image from './Image';
 
-const ImagesList = ({ isLoading, images, favorites, handleFavorite }) => {
-  const representativeImages =
-  // isLoading ?
-  //   <img className='loader' src='../assets/loader.gif' alt='Loading...' />
-  //   :
+const ImagesList = ({ images, favorites, handleFavorite }) => {
+  const representativeImages = images ?
```

Comment:
I like the use of the ternary here to display your loader if no images have been received by the component yet. And the fact that it eliminates the need for the `isLoading` prop! 🤘🏼